### PR TITLE
Updating references to the new kie-ci actions folder.

### PR DESCRIPTION
**Pull request by bacciotti**
Related PR:
- https://github.com/kiegroup/kie-ci/pull/1495

### CODE REVIEW

File: .github/workflows/pr-jenkins.yml [hunk 1]
```diff
@@ -49,8 +49,8 @@ jobs:
     name: DSL
     steps:
     - name: DSL tests
-      uses: kiegroup/kogito-pipelines/.ci/actions/dsl-tests@main
+      uses: kiegroup/kie-ci/.ci/actions/dsl-tests@main
       with:
-        main-config-file-repo: kiegroup/kogito-pipelines
+        main-config-file-repo: kiegroup/kie-ci
```
Comment (lampajr) [inline .github/workflows/pr-jenkins.yml:54]:
```suggestion
        main-config-file-repo: kiegroup/kogito-pipelines
```

File: .github/workflows/pr-jenkins.yml [hunk 2]
```diff
@@ -49,8 +49,8 @@ jobs:
...
         main-config-file-path: dsl/config/main.yaml
-        branch-config-file-repo: kiegroup/kogito-pipelines
\ No newline at end of file
+        branch-config-file-repo: kiegroup/kie-ci
```
Comment (lampajr) [inline .github/workflows/pr-jenkins.yml:56]:
```suggestion
        branch-config-file-repo: kiegroup/kogito-pipelines
```

**Review by lampajr — CHANGES_REQUESTED** [review]
@bacciotti  Meanwhile we decide if we want to keep `dsl-tests` action on `kogito-pipelines` or move it in `kie-ci`, in both cases the main and branch config file repo is still `kogito-pipelines` as this would remain the repo containing DSL configuration.

The same applies to all other repos that use dsl-tests as well

**Review by lampajr — APPROVED** [review]

**Review by Ginxo — APPROVED** [review]
LGTM

**Pull request merged**